Analyze Layout update

yungshinlintw opened the pull request:
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

hapandar committed:
Swagger changes to support automatic repairs on Virtual Machine Scale Sets (#7484)

* Update swagger to support AutomaticRepairs for VMScaleSet

* Fix unreferenced file error

* Small change - Modify the description for grace period

rifrankl committed:
Added IMDS version 2019-11-01 (#7438)

TinaHu1 committed:
[DataFactory]Add CompressionCodec property for ORC Dataset (#7452)

yungshinlintw committed:
Update for Layout Analysis

yungshinlintw committed:
Update result for layout

Commit:
Adding readme.cli.md file for peering (#7419)

* adding azure cli readme files

* updated readme.cli

* proper regexes

* fix

sandipsh committed:
Adding PolicyInsights policyStates api version 2019-10-01. (#7505)

* adding a baseline folder for PolicyInsights 2019-10-01

* fixing validation errors.

* fix go build error

* Added spec json files for other resources to be added to the package. Also updated policyStates.json as per latest API changes.

* Added new columns in the examples. Also, Removed metadata path.

xseeseesee committed:
Advisor: add readme.java.md file (#7533)

Commit:
regenerated all-api-versions

sjtuxzq committed:
[DataFactory]Add Dynamics Server-toServer authentication type. (#7437)

* Swagger changes for Dynamics S2S auth.

* Fix mistakes.

nschonni committed:
chore: jsonfmt policyinsights (#7536)

nschonni committed:
chore: jsonfmt compute (#7535)

bowen5 committed:
Add "Allocating","Upgrading" and "Compiling" enum into DeploymentResourceStatus. (#7537)

yungshinlintw committed:
Update for OCRLanguage parameter

ramparab committed:
Adding FR v2.0 API updates (#7303)

* Adding FR v2.0 API updates

* PR Validation: Fix typo, missing file ref.

* PR Validation: Fix missed typo

* PR Validation: fixing model verification errors, doc string fixes.

* Fixing model verification, added Receipts endpoints, related examples.

* Fixing semantic validation error

* Update labelFile param

* fixing types

* Fixing model validation errors

* Fixing model verification errors, other fixes.

* Fixing mode verification error.

* Fixing typo.

* Fixing doc strings, Location header update, Change response code for Analyze calls

* Incorporating feedback from Paul.

* Feedback from Florian, verification fix.

* Fixing for merge verification.

* fixing for model verification tests

* fixing model verification errors.

* fixing example file to correct useLabelFile option.

Commit:
regenerated all-api-versions

bojunehsu commented on specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json:
```diff
@@ -395,6 +395,92 @@
+          }
+        },
+        "x-ms-examples": {
+          "Analyze Layout": {
```
Minor: Analyze layout (lower case for consistency)

bojunehsu commented on specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json:
```diff
@@ -395,6 +395,92 @@
+    "/layout/analyzeResults/{resultId}": {
+      "get": {
+        "summary": "Get Analyze Layout Result",
+        "operationId": "GetAnalyzeLayoutResult",
```
"description": "Track the progress and obtain the result of the analyze layout operation.",

bojunehsu commented on specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json:
```diff
@@ -395,6 +395,92 @@
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeLayoutOperationResult"
```
Intent is for forms/receipts/layout to all use AnalyzeOperationResult instead of different output formats.

bojunehsu commented on specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json:
```diff
@@ -872,6 +958,72 @@
         }
       }
     },
+    "AnalyzeLayoutOperationResult": {
```
We don't need AnalyzeLayoutOperationResult and AnalyzeLayoutResult.

bojunehsu commented on specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/LayoutBatchResult.json:
```diff
@@ -0,0 +1,24 @@
+  "responses": {
+    "200": {
+      "body":{
+        "status": "running",
```
running would not have result.  Change to succeeded.

bojunehsu commented on specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json:
```diff
@@ -1291,6 +1443,22 @@
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "x-ms-enum": {
+        "name": "OcrDetectionLanguage",
```
Reuse Language for the x-ms-enum name so we don't generate a new enum type in the SDK.  I have verified this in SDK generation.

yungshinlintw committed:
Update from PR comment

prpandey3004 committed:
[Storage] Support private endpoint connection apis on new api version 2019-06-01 (#7440)

* [Storage] Add 2019-06-01 api version

* [Storage] Private Endpoint Connection swagger

* Added missing put privateendpoint connection definition

* PUT privateendpointconnection definition
* some refactoring/renaming
* SDK build failure due to same enum name

* Marking PrivateLinkServiceConnectionState.status as enum

* Added default error response for new apis

Commit:
regenerated all-api-versions

mathewc committed:
Updating Functions ARM API specs (#7174)

parisa-naeimi committed:
CostManagement RP- Introducing API Version 2019-10-01 plus adding Modern Commerce Account Scopes to Query and Dimension APIs.  (#7491)

* Adds base for updating Microsoft.CostManagement from version stable/2019-09-01 to version 2019-10-01

* Updates readme

* Updates API version in new specs and examples

* Updated Query and Dimension API to support GTM Scopes plus examples. Fixed an existing issue in Query API by removing value

* Fixed a small issue

* Fixed typo

* Removing unused example

* removed extra line

* Added new item for Go Package

* removed nextlink

* Fixed some minor issues

* Fixed some minor issues and also added missing description for properties

* addressed some minor issues

* added legacy vs modern

* fixed a small issue

Commit:
regenerated all-api-versions

parisa-naeimi committed:
Consumption RP - Introducing API Version 2019-10-01 with support of new Credits, lots, Event APIs, and Modern scopes for Usage Details ,and Charges APIs. (#7459)

* Adds base for updating Microsoft.Consumption from version stable/2019-06-01 to version 2019-10-01

* Updates readme

* Updates API version in new specs and examples

* Introducing new API Version 2019-10-01 with adding Credits, Lots, Events, Charges APIs and related examples

* Removed extra bracket

* Added missing bracket

* Added missing definitions

* Added missing CreditbalanceSummary

* Fixed issues for startDate and endDate

* Fixed a small issue in consumption.json

* removed GTM scopes from usage Detail

* fixed typo

* Removed charges for GTM Scope and all related examples

* Changed amount as object which includes amount plus currency

* addressed some minor issues

* Removed extra properties from credit summary

* update credit summary definition

* removed properties from creditbalancesummary

* Fixed an issue

* resolved couple of format issues

* updated description bases on ARM Suggestion

* Added support for GTM UsageDetail by introducing Kind

* fixed a small issue

* Adding Modern Scopes for Charges API with the kind support

* removed extra examples

* Updating Usage Detail for Modern

* Fixed some typos

* Updated charges examples

* Added reservation recommendation, summary, details for Modern scopes plus supporting kind for recommendation

* did some minor changes

* made some minor changes

Commit:
regenerated all-api-versions

Commit:
Fix typo in readme.go for web (#7547)

hyonholee committed:
Compute 2019-07-01 version (#7094)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* temp fix

* Adding new VirtualMachines.POST.Reapply action to compute api-version: 2019-07-01. Updating parameter info for skus api-version:2019-04-01..  (#7167)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* Adding new VirtualMachines.POST.Reapply action.

* Adding example for POST Reapply virtual machine action.

* Adding missing $filter parameter documentation to skus.json for api-version: 2019-04-01.

* Unexpected change.

* Added DiskEnryptionSet property in the managed disk parameters for the VM/VMSS (#7293)

*  updated swagger

*  updated swagger

* Removed example files which are not relevant and added links to examples for newly added property

* fixed the syntax error

*  updated swagger

*  fixed the sytax error

* Added examples for the DiskEncryptionSet property (#7318)

*  updated swagger

*  updated swagger

* Removed example files which are not relevant and added links to examples for newly added property

* fixed the syntax error

*  updated swagger

*  fixed the sytax error

*  updated code

*  udpated swagger to create DiskEncryptionSet as new property to initiate

* Resolved conflicts with DiskRP definition names for the DiskEncryptionSet (#7340)

*  updated swagger

*  updated swagger

* Removed example files which are not relevant and added links to examples for newly added property

* fixed the syntax error

*  updated swagger

*  fixed the sytax error

*  updated code

*  udpated swagger to create DiskEncryptionSet as new property to initiate

* resolved conflicts and updated swagger due to conflict in the DiskRP definitions

* Disk 2019-07-01 version (#7280)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* Add diskEncryptionSet in swagger compute-2019-07

* resolve semantic conflicts

* Fix model conflicts

* Resolve readme

* Resolve readme

* Resolve description conflicts

* Improve description

* Fix spell error

* Add some examples.

* fix model error

* Update examples (#7429)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* Add diskEncryptionSet in swagger compute-2019-07

* resolve semantic conflicts

* Fix model conflicts

* Resolve readme

* Resolve readme

* Resolve description conflicts

* Improve description

* Fix spell error

* Add some examples.

* fix model error

* Update examples

* Add extension API for VMScaleset VMs (#7407)

* add extension Api

* change operationID

* update description

* update description

* add examples

* add missing /

* add missing / in examples

* add new line

* remove unnecessary file

* add reference examples

* fix model validation failures

* remove provisioning state from params

* remove unnecessary param

* fix Delete example

* updating the description of storage account id (#7385)

* Private temp (#7456)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* Add diskEncryptionSet in swagger compute-2019-07

* resolve semantic conflicts

* Fix model conflicts

* Resolve readme

* Resolve readme

* Resolve description conflicts

* Improve description

* Fix spell error

* Add some examples.

* fix model error

* Update examples

* Remove uniqueId in swagger

* Remove required for diskEncryptionSetId

* cherry pick code owner change to compute-2019-07 (#7480)

* Iops, throughput and instance view (#7499)

* Swagger changes for IOPS, Bulk InstanceView, documentation for GETVM and InstanceView

* Fixed typos

* Updated samples

* Update 2019-07 from latest 2019-03 update.

* Update swagger (#7510)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* Add diskEncryptionSet in swagger compute-2019-07

* resolve semantic conflicts

* Fix model conflicts

* Resolve readme

* Resolve readme

* Resolve description conflicts

* Improve description

* Fix spell error

* Add some examples.

* fix model error

* Update examples

* Remove uniqueId in swagger

* Remove required for diskEncryptionSetId

* Update principalId and tenantId to be readonly

* small space fix

* add update property (#7514)

* Sync with the latest 2019-03-01 version to 2019-07-01 version of compute.

* suppress LintDiff validation error.

* Add default responses to new APIs.

Commit:
regenerated all-api-versions

asarkar84 committed:
Correcting the verbs from Create BillingProfile and InvoiceSection as per code implementation (#7524)

* Billing RP spec changes for listInvoiceSectionsWithCreateSubscriptionPermission API

* Updated the operation id

* Added x-ms-pageable

* Added next link

* Deleted unused file ProductCancelByBillingAccount

* Incorporated review comments

Incorporated review comments

* Added new properties in Invoicing APIs

Added new properties in Invoicing APIs

* Incorporated review comments

Incorporated review comments

* Added invoiceEmailOptIn in the UpdateBillingProfile request

Added invoiceEmailOptIn in the UpdateBillingProfile request

* Update billing.json

* Updating the verbs from Create BillingProfile and InvoiceSection entities

Updating the verbs from Create BillingProfile and InvoiceSection entities

sornaks committed:
Updating swagger spec to fix billing plans schema in catalogs response. (#7541)

romahamu committed:
Microsoft Azure StorageCache new API version (#7520)

* Update autorest config and swagger to support new API version 2019-11-01.

* Re-write description of some of the responses in Cache_Create.json and storagecache.json.

* Description updates.

Ready for re-review.

* Remove references to PATCH storagetarget

* Update C/ST_Create to C/ST_CreateOrUpdate

* Fix formatting.

* Fix tag name to align with previous version tags and GO SDK namespace.

ruowan committed:
update ARM email template and fix repoWhiltelist location (#7556)

andyzhangx committed:
add AKS rotateClusterCertificates API (#7436)

* add aks rotate-certs operation for 2019-08-01

* feat: use 204 instead of 200

prpandey3004 committed:
[Storage] Sync swagger changes from 2019-04-01 to 2019-06-01 apiversion (#7552)

* [Storage] Sync swagger changes from 2019-04-01 to 2019-04-01 apiversion

* Removing skiptoken which is not being used by the service

* The skiptoken is not used by the service, removing it from latest swagger.

omar-el-hariry committed:
Omelhari/feature assigning ap is (#7503)

* Adding feature assigning APIs

* Remove Pattern Feature APIs

* fix build

yangyuan committed:
Refactory CognitiveServicesAccount model specification. (#7415)

* Refactory CognitiveServicesAccount model specification.

* fix samples

zfchen95 committed:
[SRP] Add CORS to common types (#7562)

rokulka committed:
Updating the docs to include rankerType field in QueryDTO (#7538)

* Updating the docs to include rankerType field in QueryDTO

* PR comments

bleroy committed:
[Azure Search] Apply branding changes to stable/2019-05-06 and track1/stable/2019-05-06 (#7511)

* Apply branding changes to stable/2019-05-06 and track1/stable/2019-05-06

* Address code review comments

farehar committed:
Add Hybrid RP rest api spec (#7487)

* Hybrid RP swagger

* Addressed feedback

* fix typo

* fix build issues

* fix validation issues

* fix validation errors

* fix errors

* fix errors

* fix errors

* remove 204 return codes

* updated examplese

Commit:
regenerated all-api-versions

anton-evseev committed:
Merge Network's October release branch into master (#7564)

Commit:
regenerated all-api-versions

wanha-ms committed:
update-event-schema-for-azure-storage-event-grid-notification-for-cr… (#7509)

filizt committed:
[Resources] Add Deployment scripts as a new resource type - preview - (#7486)

* Approved deployments scripts swagger from private repo

* Update readme files and fix typo

* Update property letter casing

* Removing the containerSettings property due to network profile not supporting user assigned MSI.

* Update specification/resources/resource-manager/readme.csharp.md

update package name

Co-Authored-By: Yeming Liu <felix_liu@outlook.com>

Commit:
regenerated all-api-versions

bganapa committed:
Fixing issues in Azurestack admin api specs while running the powershell generator (#7566)

Commit:
fixing tag package-2019-07-01-only (#7574)

Commit:
Publish container service 2019-10-01 api version (#7567)

* Adds base for updating Microsoft.ContainerService from version stable/2019-08-01 to version 2019-10-01

* Updates readme

* Updates API version in new specs and examples

* added private FQDN (#7430)

* added private FQDN

* added example

* remove example

* updating sdk tags for 2019-10-01 api version (#7550)

* updating sdk tags for 2019-10-01 api version

* fixing ci errors

Commit:
updates for network experiment (#7573)

Commit:
scope assignmens override + flatten all (#7571)

Commit:
regenerated all-api-versions

ruowan committed:
remove 5days email notification (#7576)

Commit:
updating peering cli structure (#7572)

yungshinlintw committed:
Remove language as a parameter

yungshinlintw committed:
Merge branch 'master' of https://github.com/Azure/azure-rest-api-specs into layoutUpdate

# Conflicts:
#	specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json
